MetaMask/metamask-extension
# feat: Remove powered by blockaid line

## **Description**

Removing the "powered by Blockaid" line

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23484?quickstart=1)

## **Related issues**

Fixes: [#2196](https://github.com/MetaMask/MetaMask-planning/issues/2196)

## **Manual testing steps**

1. Launch MM
2. Open testdapp
3. Click any malicious transaction under PPOM section
4. See the powered by blockaid line
5. Checkout this branch
6. Repeat 1-3
7. See there's no powered by blockaid line

## **Screenshots/Recordings**

### **Before**
![image](https://github.com/MetaMask/metamask-extension/assets/44811/6605026d-9d35-452a-a483-72cc3246d3bf)

### **After**
<img width="363" alt="Screenshot 2024-03-18 at 17 49 55" src="https://github.com/MetaMask/metamask-extension/assets/44811/f458de06-43ca-4cf3-a3ca-ddbf23664605">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
